packages/ui-table/src/Table/README.md

```diff
@@ -310,126 +765,64 @@ class SortableTable extends React.Component {
         ))}
       </Table.Row>
     )
-  }
-
-  renderOptions () {
-    const { headers } = this.props
-    const { colTextAligns } = this.state
 
-    return (
+    const renderOptions = () => (
       <ToggleGroup
         size="small"
         toggleLabel="Set text-align for columns"
         summary="Set text-align for columns"
         background="default"
       >
-        <Table caption='Set text-align for columns'>
-          <Table.Head>
-            {this.renderHeaderRow()}
-          </Table.Head>
+        <Table caption="Set text-align for columns">
+          <Table.Head>{renderHeaderRow()}</Table.Head>
           <Table.Body>
             <Table.Row>
-              {Object.entries(colTextAligns).map(([headerId, textAlign]) => {
-                return (
-                  <Table.Cell
-                    key={headerId}
-                    width={headers.find(header => header.id === headerId).width}
```

Comment:
This width props was deleted from the original class based example and missing from the functional too. Is there a reason for that?

Reply:
According to the docs `Table.Cell` doesn't have `width` prop and also in console below warning is shown

> console.js:53 Warning: [Cell] prop 'width' is not allowed. 

That's why I removed the width prop. If it is needed for some kind of a workaround solution I can add it back.

Reply:
okay, thanks for the clarification